articles/virtual-network/security-overview.md

```diff
@@ -189,7 +189,7 @@ You can easily view the aggregate rules applied to a network interface by viewin
 
 ## Azure platform considerations
 
-- **Virtual IP of the host node**: Basic infrastructure services such as DHCP, DNS, IMDS, and health monitoring are provided through the virtualized host IP addresses 168.63.129.16 and 169.254.169.254. These IP addresses belong to Microsoft and are the only virtualized IP addresses used in all regions for this purpose.
+- **Virtual IP of the host node**: Basic infrastructure services such as DHCP, DNS, IMDS, and health monitoring are provided through the virtualized host IP addresses 168.63.129.16 and 169.254.169.254. These IP addresses belong to Microsoft and are the only virtualized IP addresses used in all regions for this purpose. Access to these IP addresses is secured through special system routes and cannot be overwritten by UDR, nor seen through the effective routes view from the Azure portal.
```

Comment:
We don't use the terms suggested on our documentation like "special" in security context has a bad optics, please modify as follows:
Effective security rules and effective routes will not include this platform rules. To override this basic infrastructure communication, you can create a security rule to Deny traffic using the following service tags on your Network Security Group rules: AzurePlatformDNS, AzurePlatformIMDS, AzurePlatformLKM.

Links to: 
https://docs.microsoft.com/en-us/azure/virtual-network/diagnose-network-traffic-filter-problem
https://docs.microsoft.com/en-us/azure/virtual-network/diagnose-network-routing-problem
https://docs.microsoft.com/en-us/azure/virtual-network/service-tags-overview